### DISCUSSION TITLE
Override page url

### DISCUSSION THREAD

**Discussion by dridi93** [squidfunk/mkdocs-material#2806] [Q&A]
Hi guys,

How can i override/rewrite page url in nav ?

eg: reference/meta-tags.md => reference/custom-url

cordially.

**Comment by squidfunk** [comment]
You can use the [`mkdocs-redirects`](https://github.com/datarobot/mkdocs-redirects) plugin.

**Reply by dridi93** [reply]
Thanks for reply,

Using [mkdocs-redirects ](https://github.com/datarobot/mkdocs-redirects) i got Redirect target '...' does not exist!

Anyway, i need just to rewrite url of existing pages...

eg, path: reference/meta-tags.md => url: reference/custom-url
